### ISSUE TITLE
Add Support for MB LAN HR

### ISSUE THREAD

**Issue by Dvaina** [bosch-thermostat/home-assistant-bosch-custom-component#324]
Hello there,
Is there any chance to get this integration supporting MB LAN HR for Ventilations (Bosch Vent 5000C)
Bosch EasyVent App works, let's say ... decent ... but an integration in Home Assistant would be nice.

I tried setting it up via IVT_MBLAN but I'm getting the following error:

`ERROR (MainThread) [bosch_thermostat_client.gateway.ivt] I cannot find supported device. Your devices: "iCom_Low_RS485_v1"`

Kind Regards

**Comment by pszafer** [comment]
Are you able to provide bosch scan?

**Comment by pszafer** [comment]
I'll reopen if you provide data needed

**Comment by Dvaina** [comment]
Sorry for the delay.

I got two bosch devices:
![image](https://github.com/bosch-thermostat/home-assistant-bosch-custom-component/assets/18976813/d9cd40f1-39c0-4a74-a513-dee8612dc760)

1. Bosh Heatpump (Gateway: ******364) connected directly to the internet and integrated via the custom component into homeassistant.
   This seems to work fine. 
   I could provide a bosch scan for this device, but I doubt that would be beneficial.
2. Bosch HVAC System "Vent 5000C" connected to the interne via an interface "MB LAN HR" (Gateway: ******296) 
  I'm trying to connect this to homeassistant via the custom component with the "IVT_MBLAN" option. Without success. I triple checked the credentials and I'm getting a different error if the enter the password incorrectly. So I doubt that's the problem.
I debugged (file attached) and the only meaningful error I could find is 
`(MainThread) [bosch_thermostat_client.gateway.ivt] I cannot find supported device. Your devices: "iCom_Low_RS485_v1"` which is the hardware version in my bosch HVAC app.
   
[home-assistant_bosch_2024-01-14T14-30-10.881Z.log](https://github.com/bosch-thermostat/home-assistant-bosch-custom-component/files/13931899/home-assistant_bosch_2024-01-14T14-30-10.881Z.log)


I suggestions what I could be doing wrong?
